.storybook/preview.js

```diff
@@ -1,8 +1,12 @@
 import { Tooltip } from '../src/directives'
 import Vue from 'vue'
+import { Translator } from './translatorMock'
+import '../../dplan/projects/diplanbau/web/css/style.0f5e11.css'
```

Comment:
this has to be removed before merging to main. And it may has to be for local testing since everyone will have other hashes.
And in the corresponding style.css all the fonts/-Path have to be removed because the path resolving is from a different spot. OR Soneohe finds out where to put the fonts, that it works

Reply:
i think we don't need to solve the styling problem within this PR. imh the styling problem must be solved by incrementally moving all needed styles to demosplan-ui, either by re-creating stuff with tailwind (after configuring tw to use our colors), or inlining complex styles within the style block of Vue components.

Reply:
Not to solve it entirely, but for local testing it would be nice to have access to the styling. That was meant.

Reply:
@ahmad-demos can You remove this before we merge it?

Reply:
@salisdemos Done in https://github.com/demos-europe/demosplan-ui/pull/379/commits/efd967ca76f89e7c59f6404d6bf1e7c1fab62db1